cmd/wasmd/root.go

```diff
@@ -189,7 +191,9 @@ func (ac appCreator) newApp(
 	var cache sdk.MultiStorePersistentCache
 
 	if cast.ToBool(appOpts.Get(server.FlagInterBlockCache)) {
-		cache = store.NewCommitKVStoreCacheManager()
+		ibCacheMetricsProvider := baseapp.MetricsProvider(cast.ToBool(viper.GetBool(server.FlagPrometheus)))
```

Comment:
In simapp, the ibCacheMetricsProvider appears to be outside of the if statement. What is the reason for this difference?

https://github.com/line/lbm-sdk/blob/v0.46.0/simapp/simd/cmd/root.go#L243

Reply:
It seems I made a mistake. 
I'll fix it. Thank you.